### PR TITLE
Add note about readthedocs build failing to docs

### PR DESCRIPTION
**Pull request by ahuang11**
image
https://github.com/spatialaudio/nbsphinx/issues/121

### CODE REVIEW

**Comment by mgeier** [comment]
Rendered: https://nbsphinx--534.org.readthedocs.build/en/534/installation.html#Jupyter-Kernel

File: doc/installation.ipynb [hunk 1]
```diff
@@ -126,7 +126,17 @@
...
     "    python3 -m pip install ipykernel\n",
     "\n",
-    "If you created your notebooks yourself with Jupyter, it's very likely that you have the right kernel installed already."
+    "If you created your notebooks yourself with Jupyter, it's very likely that you have the right kernel installed already.\n",
+    "\n",
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "Note\n",
+    "\n",
+    "If your builds on readthedocs.org are failing due to an error like the one below, add `ipykernel` to `docs/requirements.txt` to resolve.\n",
```
Comment (mgeier) [inline doc/installation.ipynb:135]: If you prepend `https:` to the URL, it will turn into a link, wouldn't that be nicer?

Comment (mgeier) [inline doc/installation.ipynb:135]: Could you please add a link to `usage.ipynb#Automatic-Creation-of-HTML-and-PDF-output-on-readthedocs.org`, where this is explained in more detail?

You could also mention `doc/environment.yml` as alternative to `docs/requirements.txt`?

Comment (mgeier) [reply doc/installation.ipynb:135]: Alternatively, you could turn the text `builds on readthedocs.org` into a link to the section mentioned above?

File: doc/installation.ipynb [hunk 2]
```diff
@@ -126,7 +126,17 @@
...
+    "\n",
+    "`jupyter_client.kernelspec.nosuchkernel: no such kernel named python3`\n",
```
Comment (mgeier) [inline doc/installation.ipynb:137]: Here you should use either an indented code block or a fenced code block.

**Review by mgeier — COMMENTED** [review]
This is great, thanks!
I just have a few minor suggestions ...

**Comment by ahuang11** [comment]
Thanks for the comments! Let me know if you have any other suggestions.

**Comment by mgeier** [comment]
Thanks a lot!

**Pull request merged**